uint16  MAX_ANGLE = 40
uint8   ID
float64 angle_in_radians

